modules/dists/BlockDist.chpl

```diff
@@ -1118,6 +1118,8 @@ proc BlockDom.doiTryCreateArray(type eltType) throws {
   // formerly in BlockArr.setup()
   if arr.doRADOpt && disableBlockLazyRAD then arr.setupRADOpt();
 
+  if !initElts then arr.dsiElementInitializationComplete();
+
```

Comment:
Why is this needed?

Reply:
I had stolen this from `setupFieldsAndAllocate` naively, but looking closer, it seems that it is already covered in `_try_ddata_allocate`, so I don't think we really need it. Will remove.